src/index.js

```diff
@@ -0,0 +1,37 @@
+// eslint-disable-next-line import/no-unresolved, import/no-absolute-path
+import './style.css';
+
+// js to access html elements
+const listDiv = document.querySelector('.to-do-list-container');
+
+// array of to do object
+const list = [
+  { description: 'was the dishes', completed: false, index: 1 },
+  { description: 'was the dishes', completed: false, index: 2 },
+  { description: 'was the dishes', completed: false, index: 3 },
+];
+
+// function to display to do list
+const displayList = function display() {
```

Comment:
- [ ] Please sort your array by index before displaying the list, as required on the project. Your code is displaying the list without considering the index field value.